packages/espresso/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
-## NEXT
+## 0.3.0
 
 * Aligns Dart and Flutter SDK constraints.
+* Changes the severity of `javac` warnings so that they are treated as errors and fixes the violations.
+* Migrates uses of the deprecated `@Beta` annotation to the new `@ExperimentalApi` annotation.
```

Comment:
This should be first, and have the `**BREAKING CHANGE**` annotation described at https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style

Reply:
Done